packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItemContextMenu.tsx

```diff
@@ -118,9 +121,13 @@ export const TreeItemContextMenu = ({
     treeItemContextMenuEntryExtensionPoint
   );
 
+  const treeItemBackendMenuContextComponents: ComponentExtension<TreeItemContextMenuComponentProps>[] = useComponents(
```

Comment:
Why backend? I'm not sure why this API has anything to do with the backend. It's just an API to let anyone contributing to the frontend to take over the behavior of an existing tree item

Reply:
renamed to `treeItemContextMenuOverrideContributions`